app/views/macros/search-results.njk

```diff
@@ -2,6 +2,54 @@
   <div>
     <h2 class="govuk-heading-s" id='totalResults' {{ model.total }} results>{{ model.total }} results</h2>
     <hr class="govubk-section-break govuk-section-reak--visible section-break-heading" role="presentation">
+    <div>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-half">
+          <p></P>
+        </div>
+        <div class="govuk-grid-column-one-half">
+          {% from "govuk/components/select/macro.njk" import govukSelect %}
+          {{ govukSelect({
+              id: "sortBySelection",
+              name: "sortBySelection",
+              href: '/results',
+              value: model.sortBy,
+              label: {
+                text: "Sort by",
+                attributes: {
+                  style: "display:contents"
+                }
+              },
+              attributes: {
+                "style": "min-width:8em",
+                "onchange": "document.forms[0].sortBy.value=this.value; document.forms[0].submit();"
```

Comment:
I would suggest using the form id instead of indexing the document.forms which could change. The searchForm has an id called 'searchForm' (very creative). So you can just use: 
`const searchForm = document.getElementById('searchForm'); searchForm.sortBy.value=this.value; searchForm.submit()` or something along those lines. I have not tested that btw.

Reply:
agreed & changed it.